app/views/shopping_shared/tabs/_contact.html.haml

```diff
@@ -0,0 +1,65 @@
+%script{ type: "text/ng-template", id: "shop/contact.html" }
+  .content#contact
+    .row
+      .small-12.large-4.columns
+        - if current_distributor.address.address1 || current_distributor.address.address2 || current_distributor.address.city || current_distributor.address.state || current_distributor.address.zipcode
+          %div.center
+            .header
+              = t :shopping_contact_address
+            %strong=current_distributor.name
+            %p
+              = current_distributor.address.address1
+              - unless current_distributor.address.address2.blank?
+                %br
+                = current_distributor.address.address2
+              %br
+              = current_distributor.address.city
+              = current_distributor.address.state
+              = current_distributor.address.zipcode
+
+      .small-12.large-4.columns
+        - if current_distributor.website || current_distributor.email_address || current_distributor.phone
+          %div.center
+            .header
+              = t :shopping_contact_web
+            %p
+              - unless current_distributor.phone.blank?
+                = current_distributor.phone
+                %br
+              - unless current_distributor.website.blank?
+                %a{href: "http://#{current_distributor.website}", target: "_blank" }
+                  = current_distributor.website
+                  %br
+              - unless current_distributor.email_address.blank?
+                %a{href: current_distributor.email_address.reverse, mailto: true}
+                  %span.email
+                    = current_distributor.email_address.reverse
+
+      .small-12.large-4.columns
+        - if current_distributor.twitter.present? || current_distributor.facebook.present? || current_distributor.linkedin.present? || current_distributor.instagram.present?
+          %div.center
+            .header
+              = t :shopping_contact_social
+            %div.follow-icons
+              - unless current_distributor.twitter.blank?
```

Comment:
this can be:
if current_distributor.twitter.present?

right?

Reply:
same for the rest of the unless statements below.

Reply:
I think these bits were just moved / indented. I'll take look now.